exercise_051_cluster_singleton_akka_bootstrap_dns/src/main/resources/application.conf

```diff
@@ -0,0 +1,79 @@
+## Recommended settings for robust cluster
+akka.cluster.shutdown-after-unsuccessful-join-seed-nodes = 30s
+akka.coordinated-shutdown.exit-jvm = on
+
+akka {
+
+  // When you're unsure about your Akka config, or you want
+  // to see what settings exist, uncomment the following line
+  //log-config-on-start = on
+
+  // We'll use Akka Cluster Bootstrap with Discovery via akka-dns
+  discovery.method = akka-dns
+
+  management.cluster.bootstrap.contact-point-discovery {
+    required-contact-point-nr = 3
+    // If you want Akka Discovery via akka-dns to use SRV records,
+    // adapt the following settings. Akka Discovery will look for
+    // SRV entries with path: _management._tcp.picluster.lb.com
```

Comment:
```suggestion
    // SRV entries with path: _management._tcp._picluster.lb.com
```
right?

Reply:
Don't think so (as verified with this config on the queries with WireShark).
The path is `_<port-name>._<protocol>.<service-name>`, so no `_` in fornt of the `service-name`.

Reply:
Interesting! If you are sure, could you update https://github.com/akka/akka/blob/master/akka-docs/src/main/paradox/discovery/index.md#discovery-method-dns accordingly?

Reply:
Will do!

Reply:
<img width="1265" alt="WireShark" src="https://user-images.githubusercontent.com/2806012/58169533-46b6d480-7c91-11e9-828d-14640c6abe49.png">